app/controllers/patients_controller.rb

```diff
@@ -281,6 +297,12 @@ def update_hoh
 
     # Change all of the patients in the household, including the current patient to have new_hoh_id as the responder
     current_user_patients.where(id: patients_to_update).each do |patient|
+      # Log the history to the new HoH if this user is still in a household
+      if patient.id == new_hoh.id && household_ids != []
```

Comment:
Did you mean for this to be `patient.responder_id`? So that every dependent would get this History item

Reply:
The PO team only wanted a message on the old HoH and new HoH without a message on the other members of the household.

Reply:
I moved this to the if block to be more consistent with how the logging is done